src/cockpit/389-console/src/lib/replication/winsyncAgmts.jsx

```diff
@@ -1291,25 +1333,35 @@ export class WinsyncAgmts extends React.Component {
                     item={this.state.agmtName}
                     checked={this.state.modalChecked}
                     mTitle="Initialize Winsync Agreement"
-                    mMsg="Are you sure you want to initialize this winsync agreement"
+                    mMsg="Are you sure you want to initialize this winsync agreement?"
```

Comment:
When I click on Initialize Winsync Agreement button - nothing happens. I still can close the confirmation modal window but 'Initialize Agreement' button doesn't work.

P.S. I've created a dummy WinSync agreement for the testing purposes

Reply:
When I close the modal window - it shows that it tries to initialize (Initializing is spinning). But it's spinning and spinning indefinitely.

Reply:
Fixed, but the endless spinning might be expected (and temporary).  It will probably timeout eventually.  For me it did not spin long but I used a invalid hostname "h" so it probably just failed right away.